pages/components/Footer.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import Image from "next/image";
+import logo from '../../public/images/logo.png';
```

Comment:
Where the images were stored was fine, since we are going to work with storybook anyways. The problem was that the path was not set correctly.